gripper/core/tasks.cpp

```diff
@@ -155,13 +164,27 @@ void gripper_tasks::QueueClient::send_capacitive_sensor_queue_rear(
     capacitive_sensor_queue_rear->try_write(m);
 }
 
+void gripper_tasks::QueueClient::send_capacitive_sensor_queue_front_isr(
+    const sensors::utils::TaskMessage& m) {
+    std::ignore = capacitive_sensor_queue_front->try_write_isr(m);
+}
+
+void gripper_tasks::QueueClient::send_capacitive_sensor_queue_rear_isr(
+    const sensors::utils::TaskMessage& m) {
+    std::ignore = capacitive_sensor_queue_rear->try_write_isr(m);
+}
+
 // gripper does not have environment nor pressure sensor
 void gripper_tasks::QueueClient::send_environment_sensor_queue(
     const sensors::utils::TaskMessage&) {}
 void gripper_tasks::QueueClient::send_pressure_sensor_queue_front(
     const sensors::utils::TaskMessage&) {}
 void gripper_tasks::QueueClient::send_pressure_sensor_queue_rear(
     const sensors::utils::TaskMessage&) {}
+void gripper_tasks::QueueClient::send_pressure_sensor_queue_front_isr(
+    const sensors::utils::TaskMessage&) {}
```

Comment:
There is no pressure sensor on the gripper so if we need this then we have made a mistake somewhere else

Reply:
Pre-existing/known issue, see line 177 comment